<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CNumberKeyboard</class>
 <widget class="QWidget" name="CNumberKeyboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>288</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>440</width>
    <height>288</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>440</width>
    <height>288</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="Key_4_but">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>72</y>
     <width>147</width>
     <height>72</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>147</width>
     <height>72</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>147</width>
     <height>72</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
background: rgb(6,17,35);
font-size:15pt;
border-top: 1px solid black;
border-left:1px solid black;  
border-bottom: 1px solid rgb(8,24,57);
border-right:1px solid rgb(8,24,57);  
color:gray;
outline: none;
} </string>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Key_7_but">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>144</y>
     <width>147</width>
     <height>72</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>147</width>
     <height>72</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>147</width>
     <height>72</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
background: rgb(6,17,35);
font-size:15pt;
border-top: 1px solid black;
border-left:1px solid black;  
border-bottom: 1px solid rgb(8,24,57);
border-right:1px solid rgb(8,24,57);  
color:gray;
outline: none;
} </string>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Key_9_but">
   <property name="geometry">
    <rect>
     <x>294</x>
     <y>144</y>
     <width>146</width>
     <height>72</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>146</width>
     <height>72</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>146</width>
     <height>72</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
background: rgb(6,17,35);
font-size:15pt;
border-top: 1px solid black;
border-left:1px solid black;  
border-bottom: 1px solid rgb(8,24,57);
border-right:1px solid rgb(8,24,57);  
color:gray;
outline: none;
} </string>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Key_2_but">
   <property name="geometry">
    <rect>
     <x>147</x>
     <y>0</y>
     <width>147</width>
     <height>72</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>147</width>
     <height>72</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>147</width>
     <height>72</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
background: rgb(6,17,35);
font-size:15pt;
border-top: 1px solid black;
border-left:1px solid black;  
border-bottom: 1px solid rgb(8,24,57);
border-right:1px solid rgb(8,24,57);  
color:gray;
outline: none;
} </string>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Key_1_but">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>147</width>
     <height>72</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>147</width>
     <height>72</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>147</width>
     <height>72</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
background: rgb(6,17,35);
font-size:15pt;
border-top: 1px solid black;
border-left:1px solid black;  
border-bottom: 1px solid rgb(8,24,57);
border-right:1px solid rgb(8,24,57);  
color:gray;
outline: none;
} </string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backspaceBtn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>216</y>
     <width>147</width>
     <height>72</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>147</width>
     <height>72</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>147</width>
     <height>72</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
background: rgb(6,17,35);
font-size:15pt;
border-top: 1px solid black;
border-left:1px solid black;  
border-bottom: 1px solid rgb(8,24,57);
border-right:1px solid rgb(8,24,57);  
color:gray;
outline: none;
} </string>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Key_3_but">
   <property name="geometry">
    <rect>
     <x>294</x>
     <y>0</y>
     <width>146</width>
     <height>72</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>146</width>
     <height>72</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>146</width>
     <height>72</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
background: rgb(6,17,35);
font-size:15pt;
border-top: 1px solid black;
border-left:1px solid black;  
border-bottom: 1px solid rgb(8,24,57);
border-right:1px solid rgb(8,24,57);  
color:gray;
outline: none;
} </string>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Key_6_but">
   <property name="geometry">
    <rect>
     <x>294</x>
     <y>72</y>
     <width>146</width>
     <height>72</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>146</width>
     <height>72</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>146</width>
     <height>72</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
background: rgb(6,17,35);
font-size:15pt;
border-top: 1px solid black;
border-left:1px solid black;  
border-bottom: 1px solid rgb(8,24,57);
border-right:1px solid rgb(8,24,57);  
color:gray;
outline: none;
} </string>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Key_5_but">
   <property name="geometry">
    <rect>
     <x>147</x>
     <y>72</y>
     <width>147</width>
     <height>72</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>147</width>
     <height>72</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>147</width>
     <height>72</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
background: rgb(6,17,35);
font-size:15pt;
border-top: 1px solid black;
border-left:1px solid black;  
border-bottom: 1px solid rgb(8,24,57);
border-right:1px solid rgb(8,24,57);  
color:gray;
outline: none;
} </string>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Key_0_but">
   <property name="geometry">
    <rect>
     <x>147</x>
     <y>216</y>
     <width>147</width>
     <height>72</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>147</width>
     <height>72</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>147</width>
     <height>72</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
background: rgb(6,17,35);
font-size:15pt;
border-top: 1px solid black;
border-left:1px solid black;  
border-bottom: 1px solid rgb(8,24,57);
border-right:1px solid rgb(8,24,57);  
color:gray;
outline: none;
} </string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Key_8_but">
   <property name="geometry">
    <rect>
     <x>147</x>
     <y>144</y>
     <width>147</width>
     <height>72</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>147</width>
     <height>72</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>147</width>
     <height>72</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
background: rgb(6,17,35);
font-size:15pt;
border-top: 1px solid black;
border-left:1px solid black;  
border-bottom: 1px solid rgb(8,24,57);
border-right:1px solid rgb(8,24,57);  
color:gray;
outline: none;
} </string>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Key_point_but">
   <property name="geometry">
    <rect>
     <x>294</x>
     <y>216</y>
     <width>146</width>
     <height>72</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>146</width>
     <height>72</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>146</width>
     <height>72</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
background: rgb(6,17,35);
font-size:15pt;
border-top: 1px solid black;
border-left:1px solid black;  
border-bottom: 1px solid rgb(8,24,57);
border-right:1px solid rgb(8,24,57);  
color:gray;
outline: none;
} </string>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
